src/modules/FioAddress/util.js

```diff
@@ -16,9 +16,10 @@ const CONNECTED_WALLETS = 'ConnectedWallets.json'
 const FIO_ADDRESS_CACHE = 'FioAddressCache.json'
 const FIO_EXPIRED_CHECK = 'FioExpiredCheck.json'
 const MONTH = 1000 * 60 * 60 * 24 * 30
-const BUNDLES_AMOUNT_ALERT = 5
 const DEFAULT_BUNDLE_SET_VALUE = 1
 
+export const BUNDLES_AMOUNT_ALERT = 5
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name